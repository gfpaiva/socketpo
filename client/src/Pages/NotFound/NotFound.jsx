import React from 'react';

const NotFound = () => (
	<h1>
		Page not foud <span role="img" aria-label="Neutral Face">😐</span>
	</h1>
);

export default NotFound;
