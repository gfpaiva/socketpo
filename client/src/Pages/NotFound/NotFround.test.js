import React from 'react';
import { mount } from 'enzyme';
import NotFound from './NotFound';

describe('<NotFound />', () => {

	it('should mount properly', () => {

		const wrapper = mount(<NotFound />);

		expect(wrapper.find('NotFound')).toMatchSnapshot();
	});
});
