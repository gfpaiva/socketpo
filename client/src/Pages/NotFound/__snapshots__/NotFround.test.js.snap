// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should mount properly 1`] = `
<NotFound>
  <h1>
    Page not foud 
    <span
      aria-label="Neutral Face"
      role="img"
    >
      😐
    </span>
  </h1>
</NotFound>
`;
